src/app/plugins/pdfjs-dist.ts

```diff
@@ -7,7 +7,7 @@ export const usePdfJSLoader = () =>
   useAsyncCallback(
     useCallback(async () => {
       const pdf = await import('pdfjs-dist');
-      pdf.GlobalWorkerOptions.workerSrc = '/pdf.worker.min.js';
+      pdf.GlobalWorkerOptions.workerSrc = 'pdf.worker.min.js';
```

Comment:
As scripts are served from `/assets` i wonder where will it pick pdf.worker.min.js (its in `rootPath/`). `rootPath/pdf.worker.min.js` or `rootPath/assets/pdf.worker.min.js`?

Reply:
for visibility: https://github.com/cinnyapp/cinny/pull/1603/commits/845eebe98af4405bb0840b74d538b5c64522a676 adds build config for providing base url so we can also do like https://github.com/cinnyapp/cinny/pull/1603/commits/845eebe98af4405bb0840b74d538b5c64522a676#diff-fd463d18da0dee07090abb23a56e8e05cf2113a48d1820cb013a5b63d3abd2f9

Reply:
After #1603 I need to change `build.config.ts` to include my sub path for index.js to load at all.
Then `pdf.worker.min.js` is loaded absolute from the url. Not absolute to relative path.

Using `${trimTrailingSlash(import.meta.env.BASE_URL)}/pdf.worker.min.js` fixes the problem too.

You decide. I think removing the slash is simpler than trimming and reading env.
`pdf.worker.min.js` is never loaded from `assets/`, it is not even available there